lang/en/lang.php

```diff
@@ -219,5 +219,27 @@
         'redirect_desc' => 'Page name to redirect if access is denied.',
         'logout' => 'You have been successfully logged out!',
         'stop_impersonate_success' => 'You are no longer impersonating a user.',
-    ]
+    ],
+    'reportwidgets' => [
+        'statistics' => [
+            'default_title' => 'Plugin User - Statistics',
+            'registration' => 'Registration',
+            'activated' => 'Activated',
+            'ban' => 'Ban',
+            'today' => 'today',
+            'yesterday' => 'yesterday',
+            'month' => 'month',
+            'year' => 'year',
```

Comment:
```suggestion
            'today' => 'Today',
            'yesterday' => 'Yesterday',
            'month' => 'This month',
            'year' => 'This year',
```

Reply:
@bennothommo you indicated how: this month, this year.
But that's not what I meant. Not this month / year, but the last 30/356 days.
That is, today is November 19, 2021.
`Month` - we take statistics from October 19 to November 19.
`Year` - statistics from November 19, 2020 to November 19, 2021

Reply:
There is no error handling in my code. I'll fix it.